### PR TITLE
Split out "sh" from build requirments

### PR DESCRIPTION
**Pull request by tomdee**
It can't be installed on Windows

### CODE REVIEW

File: build-requirements-nosh.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+docopt==0.6.2
```
Comment (caseydavenport) [inline build-requirements-nosh.txt:1]: "nosh" ?

https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=nosh

Comment (caseydavenport) [inline build-requirements-nosh.txt:1]: I guess this is food for the `build-requirements.txt` file?

**Comment by caseydavenport** [comment]
Failing tests, and I'm not the biggest fan of the word "nosh", but I guess I can let that slide....

**Comment by tomdee** [comment]
Closing in favor of #94

**Pull request closed without merging**